file-formats/scp1/example/z_aff_example_scp1.scp1.json

```diff
@@ -0,0 +1,1202 @@
+{
```

Comment:
this file is located in the wrong folder (`example`). It should be `examples`.

This is als the reason why the link check raises an issue

Reply:
Thank you for the hint. All checks have passed, you can start the review.